src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,189 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.EmptyTaskDescriptionException;
+import duke.exceptions.EmptyTaskNumberException;
+import duke.exceptions.InvalidTaskException;
+import duke.exceptions.MultipleKeywordException;
+import duke.exceptions.NoTimeException;
+import duke.exceptions.TaskDoneAlreadyException;
+import duke.exceptions.UnknownInputException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.util.DukeCommands;
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
```

Comment:
This is a large group of imports, perhaps consider separating imports of similar subclasses? (e.g. have a line break between duke.exceptions and duke.task and so on.)

Reply:
I tried that, but it seems to cause a checkstyle violation. I think imports from the same package are all grouped together